README.md

```diff
@@ -125,6 +125,8 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 
 ## Open Source books
 - [Producing Open Source Software](https://producingoss.com/) - Producing Open Source Software is a book about the human side of Open Source development. It describes how successful projects operate, the expectations of users and developers, and the culture of free software.
+- [Pro Git (Second Edition)](https://www.freetechbooks.com/pro-git-second-edition-t1202.html) - A fully-updated guide to Git and its usage in the modern world. This book teaches readers how to master the world of distributed version workflow, use the distributed features of Git to the full, and extend Git to meet their every need.
```

Comment:
This book is already freely available in the previous section on Using Version Control https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/README.md#L117

```suggestion
```

So we can just remove this.